vendor/brave-ios/BUILD.gn

```diff
@@ -19,6 +20,7 @@ config("internal_config") {
     "-Wno-objc-property-synthesis",
     "-Wno-sign-compare",
   ]
+  ldflags = [ "-Wl,-no_compact_unwind" ]
```

Comment:
this is needed for rust, correct? If so, we should add a rust config and do `configs +=` so we can better track what these are used for